### PR TITLE
Add support for CelOS

### PR DESCRIPTION
**Pull request by MrVeink**
#123 Adding CelOS ASCII art. Tested by editing /etc/os-release on openSUSE Tumbleweed. I also checked the ASCIIs above and below to make sure I didn't break em.

### CODE REVIEW

**Comment by MrVeink** [comment]
@whyadamsalt I hope it's ok that the lines are white instead of black. I made it so because of the terminal being black.

**Comment by whyadamsalt** [comment]
> @whyadamsalt I hope it's ok that the lines are white instead of black. I made it so because of the terminal being black.

Should be fine! Thank you!

**Comment by MrVeink** [comment]
> > @whyadamsalt I hope it's ok that the lines are white instead of black. I made it so because of the terminal being black.
> 
> Should be fine! Thank you!

Ofc. I got bored so I was going thru the Issues list and thought this might be an easy one. I tried first to use an online photo to ASCII website but it didn't look that good so I made this one from scratch.

**Comment by whyadamsalt** [comment]
> > > @whyadamsalt I hope it's ok that the lines are white instead of black. I made it so because of the terminal being black.
> 
> > 
> 
> > Should be fine! Thank you!
> 
> 
> 
> Ofc. I got bored so I was going thru the Issues list and thought this might be an easy one. I tried first to use an online photo to ASCII website but it didn't look that good so I made this one from scratch.

Well thank you for all of your hard work!

**Comment by yuri-norwood** [comment]
Is there a reason that the LICENSE, README, and Makefile are being changed to executable?

**Comment by MrVeink** [comment]
> Is there a reason that the LICENSE, README, and Makefile are being changed to executable?

What do you mean?

**Comment by yuri-norwood** [comment]
> > Is there a reason that the LICENSE, README, and Makefile are being changed to executable?
> 
> What do you mean?

The file permissions have changed from `655` to `755`, did you run `chmod`?

image

File: pfetch [hunk 1]
```diff
@@ -465,7 +465,7 @@ get_host() {
         case $word in
            (To      | [Bb]e      | [Ff]illed | [Bb]y  | O.E.M.  | OEM  |\
             Not     | Applicable | Specified | System | Product | Name |\
-            Version | Undefined  | Default   | string | INVALID | �    | os |\
+            Version | Undefined  | Default   | string | INVALID | ?    | os |\
```
Comment (dylanaraps) [inline pfetch:468]: Remove this change.

**Comment by MrVeink** [comment]
> > > Is there a reason that the LICENSE, README, and Makefile are being changed to executable?
> > 
> > What do you mean?
> 
> The file permissions have changed from `655` to `755`, did you run `chmod`?
> 
> image
> 

Oh lmao didn't realize. Just a mistake.

File: pfetch (hunk 1 above)
Comment (MrVeink) [reply pfetch]: May I ask what's the difference?

File: pfetch [hunk 2]
```diff
@@ -1181,6 +1181,19 @@ get_ascii() {
...
+				${c5}     //_         \\\\
+				${c5}    /_  ${c7}##############
+				${c5}   //              *\\
+				${c7}###############    ${c5}|#
+				${c5}   \/              */
+				${c5}    \*   ${c7}##############
+				${c5}     */,        .//
+				${c5}      '_///\\\\\//_'
+			EOF
+        ;;
```
Comment (dylanaraps) [inline pfetch:1196]: Add a newline here (below `;;`).

**Pull request merged**